### PR TITLE
Adding LICENSE

### PR DESCRIPTION
**Pull request by trantienduchn**
I tested by create LICENSE file at new local branch, then creating PR to merge. But after merging, it doesn't show Apache license at github page. Then I came to create License file by [this guideline](https://help.github.com/articles/adding-a-license-to-a-repository/) and It works on my local repository

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (blackheaven) [inline LICENSE]: Something have to be filled here

Comment (chibenwa) [reply LICENSE]: Some projects like https://github.com/apache/jmeter did not include that section. From a short analysis looking Apache project practices, this seems to be **very** common hence I propose we also do the same.

(I could not find counter-exemples...)

Comment (blackheaven) [reply LICENSE]: I do not mind if it is removed, as long nothing remains unfilled.

Comment (chibenwa) [reply LICENSE]: That's the point: is the license still recognised by github?

Seems fair to be doing like others for this.

Comment (blackheaven) [reply LICENSE]: I think so: https://github.com/apache/cloudstack/blob/master/LICENSE

Comment (trantienduchn) [reply LICENSE]: let's try

**Comment by aduprat** [comment]
Was :green_apple: before restarting the CI.

**Comment by openpaas-james** [comment]
Build triggered. sha1 is merged.

**Comment by openpaas-james** [comment]
Build started sha1 is merged.

**Comment by openpaas-james** [comment]
Build finished. No test results found.

**Comment by chibenwa** [comment]
test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**